modules/simple-payments/simple-payments.css

```diff
@@ -26,7 +26,7 @@ body .jetpack-simple-payments-wrapper .jetpack-simple-payments-details p {
 }
 
 /* Higher specificity in order to trump theme's style */
-body .jetpack-simple-payments-wrapper .jetpack-simple-payments-product-image .jetpack-simple-payments-image img.size-full {
+body .jetpack-simple-payments-wrapper .jetpack-simple-payments-product-image .jetpack-simple-payments-image img {
```

Comment:
Could you explain a bit the reasons behind this change?

Reply:
This CSS rule ensures the image fits into the `jetpack-simple-payments-image` wrapper defined above. However, it was previously applied only to images with the `size-full` class, which was [added by `get_the_post_thumbnail` when parsing the shortcode](https://github.com/Automattic/jetpack/blob/b8648d7ca8f3d9209f3b6f95fe00a2e760af7876/modules/simple-payments/simple-payments.php#L230).

[We no longer use `get_the_post_thumbnail` in the new `save` function](https://github.com/Automattic/jetpack/blob/a905aa011d984ddb74021908dc2cc5fb7dda07a7/extensions/blocks/simple-payments/save.js#L33) (which means the `size-full` class is no longer added), so this change makes sure the image keeps taking the size of the wrapper.

Also, if for any reason we change in the future the `get_the_post_thumbnail` call to use a different image size when rendering the shortcode, it should keep taking the size of the wrapper regardless the size chosen.